routers/api/v1/api.go

```diff
@@ -675,7 +675,7 @@ func mustEnableWiki(ctx *context.APIContext) {
 
 func mustNotBeArchived(ctx *context.APIContext) {
 	if ctx.Repo.Repository.IsArchived {
-		ctx.NotFound()
+		ctx.Error(http.StatusLocked, "RepoArchived", fmt.Errorf("repo %s/%s is archived", ctx.Repo.Repository.OwnerName, ctx.Repo.Repository.Name))
```

Comment:
I think you can use `LogString` here as well.
However, I don't know if on `Repo` or on `Repository`.